index.html

```diff
@@ -5,39 +5,167 @@
+            </a>
+          </div>
+
+        <!--- Headline Section --->
+          <div class="headline">
+            <div class="headline-text1">Hello, I am Ben.</div>
+            <div class="headline-text2">The fullstack developer you didn't know you needed.</div>
+            <div class="headline-details">Discover my work and contact me to reach the sky together</div>
+            <div class="social">
+              <div class="let-connect">Let's Connect</div>
 
-    <!--- Top navigation Bar --->
-    <div class="topnav">
-      <div class="left"></div>
-      <div class="nav-menu">
-        <a href="#news">Portfolio</a>
-        <a href="#about">About</a>
-        <a href="#contact">Contact</a>
+              <!-- Add font awesome icons -->
+              <ul class="icon-list">
+                <a href="https://facebook.com/benj.p009" target="_blank" rel="noopener" class="fa fa-facebook"></a>
+                <a href="https://instagram.com/benjamin_patin" target="_blank" rel="noopener" class="fa fa-instagram"></a>
+                <a href="https://linkedin.com/in/benjaminpatin" target="_blank" rel="noopener" class="fa fa-linkedin"></a>
+                <a href="https://twitter.com/benjp009" target="_blank" rel="noopener" class="fa fa-twitter"></a>
+                <a href="https://github.com/benjp009" target="_blank" rel="noopener" class="fa fa-github"></a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
       </div>
-      <div class= "right"></div>
-    </div>
 
-  <!--- Headline Section --->
-    <div class="headline">
-      <div class="headline-text1">Hey There, I am Benjamin Patin.</div>
-      <div class="headline-text2">I am a Full Stack developper.</div>
-      <div class="headline-details"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-don’t hestiate to contact me.</div>
-      <div class="social">
+      <!-- The "works" categories are here -->
+      <div class="grid-container">
+        <div class="grid-item">
+            <img class="work-section-image" src="ressources/Work-1.png" alt="Picture for the first work of Benjamin Patin">
+            <h2>Tonic</h2>
+            <ul class="client-info">
+              <li class="client-name">CANOPY</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-style">Back End Dev</li>
+              <img class="client-section-counter" src="ressources/Counter.png" alt="Counter">
+              <li class="project-year">2015</li>
+            </ul>
+            <p class="project-info">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="language-used">
+              <div class="tags">html</div>
+              <div class="tags">css</div>
+              <div class="tags">javaScript</div>
+          </div>
```

Comment:
- [ ] Let's change this `div` for an `ul` tag with `li` tags inside. Since this semantically a list of elements. Remember to do it in each card. :slightly_smiling_face: